doc/release-notes/9983-unique-constraints.md

```diff
@@ -0,0 +1,14 @@
+This release adds two missing database constraints that will assure that the externalvocabularyvalue table only has one entry for each uri and that the oaiset table only has one set for each spec. (In the very unlikely case that your existing database has duplicate entries now, install would fail. This can be checked by running
+
+SELECT uri, count(*) FROM externalvocabularyvaluet group by uri;
+
+and
+
+SELECT spec, count(*) FROM oaiset group by spec;
+
+and then removing any duplicate rows (where count>1).
+
+
+
+
+TODO: Add note about reloading metadata blocks after upgrade.
```

Comment:
is this a todo that we want before we qa / merge this?

Reply:
I think that is just a note that the main release notes should include the reload block instructions for these blocks.

Reply:
Do you guys think we should clarify that whoever makes the release notes has to do this? I am not sure it is clear enough with the **_TODO:_** probably we could just mention it on Slack but we run the risk of someone missing the message 🤷🏼 @scolapasta @qqmyers